src/notifications/notificatielog/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import include, path
+
+from .views import NotificatieLogListView, NotificatieResponseLogListView
+
+app_name = 'Notificatie Log'
```

Comment:
weird name - this is used in reversing urls and acts as namespace
```suggestion
app_name = 'notificatielog'
```